README.md

```diff
@@ -76,28 +76,31 @@ For detailed instructions for different varieties of Linux see [LINUX\_INSTALL.m
 For common installation issues see [INSTALL\_ISSUES.md](INSTALL_ISSUES.md).
 
 We recommend using virtualenv to use, install, or build Turi Create. 
-Be sure to install virtualenv using your system pip.
 
 ```shell
 pip install virtualenv
 ```
 
 The method for installing *Turi Create* follows the
 [standard python package installation steps](https://packaging.python.org/installing/).
-To create a Python virtual environment called `venv` follow these steps:
+To create and activate a Python virtual environment called `venv` follow these steps:
 
 ```shell
 # Create a Python virtual environment
 cd ~
 virtualenv venv
-```
 
-To activate your new virtual environment and install `Turi Create` in this environment, follow these steps:
-```shell
 # Activate your virtual environment
 source ~/venv/bin/activate
+```
+Alternatively, if you are using Anaconda, you may use its virtual environment:
```

Comment:
I'd link to [Anaconda](https://www.anaconda.com/what-is-anaconda/) here.

Reply:
Done